hawtio-war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -105,6 +105,10 @@
     <filter-name>AuthenticationFilter</filter-name>
     <url-pattern>/jolokia/*</url-pattern>
   </filter-mapping>
+  <filter-mapping>
+    <filter-name>AuthenticationFilter</filter-name>
+    <url-pattern>/proxy/*</url-pattern>
+  </filter-mapping>
```

Comment:
`<url-pattern>` should be able to be defined multiple times in a single `<filter-mapping>`.

Reply:
oh my - I forgot ;)